### PR TITLE
add pmc and maintainer list

### PR DESCRIPTION
**Pull request by gingerkidney**

### CODE REVIEW

File: architecture/maintainer-list.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Maintainer List
+
+|                GitHub ID                |  Name  | Company |
+| :-------------------------------------: | :----: | :-----: |
+|   [shenli](https://github.com/shenli)   |  申砾  | PingCAP |
+|  [jackysp](https://github.com/jackysp)  | 于帅鹏 | PingCAP |
+|  [coocood](https://github.com/coocood)  | 周昱行 | PingCAP |
+|  [winkyao](https://github.com/winkyao)  |  姚维  | PingCAP |
+| [zz-jason](https://github.com/zz-jason) | 孙晓光 | PingCAP |
```
Comment (dcalvin) [inline architecture/maintainer-list.md]:
```suggestion
| [zz-jason](https://github.com/zz-jason) | 孙晓光 | Zhihu |
```

Comment (gingerkidney) [reply architecture/maintainer-list.md]: the last maintainer should be zhangjian, I fixed the wrong name

File: architecture/README.md [hunk 2]
```diff
@@ -33,7 +33,9 @@ types of roles. The TiDB community has the following organizations:
 
 ## PMC
```
Comment (dcalvin) [inline architecture/README.md]: @gingerkidney Suggesting removing this file, as it conflicts with https://github.com/pingcap/community/pull/152

Comment (zz-jason) [reply architecture/README.md]: I think we'd better keep a `README.md` file for a directory.

Comment (dcalvin) [reply architecture/README.md]: Sorry, I mean revert/reset this change in this file for this PR. Not removing it...

**Pull request closed without merging**